docs/Azure_IoT_Integration_Guide/Azure_IoT_Central_integration/Set_an_Observation.md

```diff
@@ -0,0 +1,150 @@
+
+4. Enter a name for your device template and click **Next: Review**. In this example, our device template is called `watercapacity`.
+
+    ![Create new device template](images-observation/observation-central3.png "Enter a name for your new device template")
+
+5. On the **Review** page, click **Create**.
+
+6. On the **Create a model** page, select the **Import a model** tile. This is where you need to import the .JSON file you exported from Coiote DM.
+
+7. On the **Model** page, click **Publish** to publish this device template to the application. In the pop-up window that appears, click **Publish**.
+
+### Assign a device template in Azure IoT Central
+1. To assign the published template to a device, go to **Devices** from the left pane and click on the device you want to observe. In our case, the device is called `soilquality`.
+
+2. Click **Manage templates** and choose **Assign template**.
+
+    ![Assign a template](images-observation/observation-central4.png "Assign a device template")
+
+3. In the pop-up window that appears, find the template you have previously created in Azure (in our case, this is `watercapacity`) and click **Assign template**.
+
+## Create a group
+
+Now that you have a device with a template assigned to it, you need to add this device to a group. To create a group:
+
+1. In Azure IoT Central, go to **Device groups** from the left pane and click **+ New**.
+
+    !!! note
+        A device group can only contain devices from a single device template.
+
+2. Fill in the following information:
+    * **Group name** and an optional description. In our case, the group is called `soil`.
+    * **Device group access**: skip, because Lwm2m test application doesn’t use organizations.
+    * **Create a device query**: from the **Value** drop-down menu, select the name of the device template you created in the previous step (`watercapacity` in our case). Click **Run query**.
+
+After running a query, you will see the list of devices that are associated with this template. In our case it’s a device called `soilquality`. Click **Save**.
+
+![Create a device query](images-observation/observation-central5.png "Create a device query")
+
+## Set up an Observation
+
+Let’s set an Observation operation on resource ID **6/0/2**. This is Altitude, and the data is sent as Telemetry.
+
+To add an Observation, you need to use the **Jobs** tab.
+
+1. In Azure IoT Central, go to **Jobs** from the left pane and click **+ New job**.
+
+2. Configure your job:
+   * **Name ***: Provide a name and an optional description of the job.
+   * **Access**: skip, because Lwm2m test application doesn’t use organizations.
+   * **Target devices**: choose which devices this job will run on.
+   * **Job properties**:
+       * Set **Job type** as `Property`.
+       * Under the **Name** field, select the resource you want to put the observation for and turn the toggle to **True**.
```

Comment:
jeśli mamy tutaj podejście tutorialowe, to ja bym wymienił z nazwy resource, o który nam chodzi (wiem, że jest na screenie, ale imo to nie wystarcza). Podobnie poniżej z parametrem

Reply:
dzięki, dodałam to